src/libs/util/statement.cpp

```
@@ -0,0 +1,123 @@
+    if (res == SQLITE_ROW) {
+        return true;
+    }
+
+    if (res != SQLITE_DONE) {
+        m_lastError = QString(static_cast<const QChar *>(sqlite3_errmsg16(sqlite3_db_handle(m_stmt))));
+    }
+
+    return false;
+}
+
+QVariant Statement::value(int index) const
```

Cppcheck (2.20.0)

[style] The function 'value' is never used.

(unusedFunction)